### PR TITLE
1334755: Stack Trace on Identical Manifest

### PR DESCRIPTION
**Pull request by pjonastik**
Removal of stack trace logging on identical manifest
import.

### CODE REVIEW

File: server/src/main/java/org/candlepin/resource/OwnerResource.java [hunk 1]
```diff
@@ -1256,21 +1257,31 @@ else if (overrideConflicts[0].equalsIgnoreCase("false")) {
...
         }
         catch (ImporterException e) {
+            log.error("Recording import failure", e);
             recordImportFailure(owner, data, e, filename);
             throw new IseException(e.getMessage(), e);
         }
+        catch (ImportConflictException e) {
+            //This is a normal path of operation
+            //and does not require a stack trace in the log.
+            recordImportFailure(owner, data, e, filename);
```
Comment (barnabycourt) [inline server/src/main/java/org/candlepin/resource/OwnerResource.java]: Having a debug log statement here would be good.

**Comment by barnabycourt** [comment]
@pjonastik You have 3 identical error messages for different exceptions. Can more detail be provided in the log message to help the user resolve the problem?

**Comment by pjonastik** [comment]
I missed that messages. Now should be correct.

**Comment by barnabycourt** [comment]
@pjonastik It is still missing a debug log statement for the ImportConflictException

**Comment by barnabycourt** [comment]
@pjonastik the two commits should be squashed to a single commit

**Comment by pjonastik** [comment]
@barnabycourt What do you meant by last comment? I used commit with "-amend" option, so it should be squashed in one commit.

**Comment by barnabycourt** [comment]
@pjonastik it looks good as one commit. Still waiting on the debug statement for the ImportConflictException

**Comment by pjonastik** [comment]
@barnabycourt I added debug statement.

**Comment by nguyenfilip** [comment]
@pjonastik pleaes fix the commit message (title). It should be "Bugzilla numer: ....." so not " 1334755 - Stack Trace on Identical Manifest " but "1334755: Stack Trace on Identical Manifest"

**Comment by nguyenfilip** [comment]
retest this please

**Pull request merged**